openedx/features/caliper_tracking/tests/expected/seek_video.json

```diff
@@ -0,0 +1,50 @@
+{
+    "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+    "action": "Paused",
```

Comment:
`Paused` is not a suitable option for this. Please use another appropriate `action`. I believe `JumpedTo` is appropriate in this case.

Reply:
Oh it was a mistake during updating Json, it is `JumpedTo`.